test/gurka/test_exclude_unpaved_smoothness.cc

```diff
@@ -146,7 +146,8 @@ TEST_F(ExcludeUnpavedTest, UnpavedRoadsInTheEnd) {
   }
 }
 
-valhalla::gurka::map BuildPBF(const std::string& workdir) {
+TEST(Standalone, SmoothnessAccess) {
```

Comment:
Restructure this test to be more idiosyncratic (i.e. use gurka helper methods instead of calling the `build_tile_set` directly).